src/main/scala/circt/stage/phases/CIRCT.scala

```diff
@@ -163,7 +160,6 @@ class CIRCT extends Phase {
         (!inferReadWrite).option("-disable-infer-rw") ++
         (!imcp).option("-disable-imcp") ++
         /* The following options are off by default, so we enable them if they are true. */
-        (dedup).option("-dedup") ++
```

Comment:
This should be `Some("-dedup")`. Dedup is off by default and we want to guarantee it's on.